src/msw/overlay.cpp

```diff
@@ -162,7 +162,18 @@ void wxOverlayImpl::Init(wxDC* dc, int , int , int , int )
 
     // The rectangle must be in screen coordinates
     m_rect = m_window->GetClientSize();
-    m_window->ClientToScreen(&m_rect.x, &m_rect.y);
+
+    if ( m_window->GetLayoutDirection() == wxLayout_LeftToRight )
+    {
+        m_window->ClientToScreen(&m_rect.x, &m_rect.y);
+    }
+    else // wxLayout_RightToLeft
+    {
+        RECT rc;
+        wxCopyRectToRECT(m_rect, rc);
+        wxMapWindowPoints(GetHwndOf(m_window), HWND_DESKTOP, &rc);
+        wxCopyRECTToRect(rc, m_rect);
+    }
 
```

Comment:
See this [answer](https://stackoverflow.com/questions/15734528/client-rectangle-coordinates-on-screen#comment22378286_15734569) by [Raymond Chen](https://stackoverflow.com/users/902497/raymond-chen)